### ISSUE TITLE
Error: Couldn't find corresponding reader for BUTTON

### ISSUE THREAD

**Issue by ToonTalk** [NetLogo/Galapagos#310]
This happens with http://www.netlogoweb.org/launch# and not http://li425-91.members.linode.com:9000/create-standalone

for http://764-dot-m4a-gae-hrd.appspot.com/p/qHXw4eFM0LmLNxc47rDE66.nlogo

This happens with "Choose File"

**Comment by TheBizzle** [comment]
I was able to reproduce the bad behavior with the original model.  However, when I opened it in NetLogo 5.2.0 (on my Linux machine), saved the model again, and tried to load it in NetLogo Web again, it worked.

But the model claims to have come from NetLogo 5.2.0, so I think something strange is going on here.  I checked the diffs between the original and the version I resaved, and I found them really weird—maybe an issue with newlines...?

@ToonTalk, did you originally save this `.nlogo` file on Windows?  Or... how was it created?
